tasks/create.yaml

```diff
@@ -16,7 +17,7 @@ tasks:
       - task: pepr-build
 
       - description: "Create the UDS Core Standard Zarf Package"
-        cmd: "uds zarf package create packages/standard --confirm --no-progress --flavor ${FLAVOR}"
+        cmd: "uds zarf package create packages/standard --confirm --output .github/build/${K8S_DISTRO} --no-progress --flavor ${FLAVOR}"
```

Comment:
I suspect this addition is what is causing CI failures at the moment - our other tasks for k3d CI assume that created packages are under `build/` at the root. Is there a reason these have to be moved to `.github` and placed per distro? If so, we could probably use maru `if` lines in the task here and conditionally add the `--output` flag.

Reply:
Ah good callout. I'll fix that.

Reply:
I think that works - I'm still a bit unclear on why a different build folder is needed for eks vs rke2? The core package layer should be standard between them (and k3d) so couldn't we just continue to point to `../../../build` and not pass in a specific output path? In other words, the only unique part between rke2, eks, and k3d is the bundle, but all should be able to import the same package for core.

Reply:
Ahh, I see. You're 100% right. I'll back those changes out.

Reply:
Np - just figured we could potentially reduce the added var/flag here, but wasn't sure if I missed something.